cmd/lncli/commands.go

```diff
@@ -2913,11 +2913,40 @@ func describeGraph(ctx *cli.Context) error {
 var listPaymentsCommand = cli.Command{
 	Name:     "listpayments",
 	Category: "Payments",
-	Usage:    "List all outgoing payments.",
+	Usage:    "Query the payment history.",
```

Comment:
List payments includes payments that are still in flight so 'history' doesn't quite parse.

Reply:
Reverted this change.